Library/Homebrew/linux_runner_spec.rb

```diff
@@ -8,25 +8,4 @@ class LinuxRunnerSpec < T::Struct
   const :workdir, String
   const :timeout, Integer
   const :cleanup, T::Boolean
-
-  sig {
-    returns({
-      name:      String,
-      runner:    String,
-      container: T::Hash[Symbol, String],
-      workdir:   String,
-      timeout:   Integer,
-      cleanup:   T::Boolean,
-    })
-  }
-  def to_h
```

Comment:
Now I remember why I went with `#to_h`:

> **However, `serialize` and especially `from_hash` are particularly fraught** (see the “gotchas” sections below). It’s likely better to do manual conversion to and from Hash values

https://sorbet.org/docs/tstruct#serialize-and-from_hash-converting-tstruct-to-and-from-hash

Reply:
Ah, ok, I'm going to close this out then. I thought maybe `serializs` wasn't considered, but I trust you made a considered decision in that case. Thanks!

Reply:
We can still get rid of the type aliases though.